File: src/main/java/com/example/matchteam/dto/board/comment/EnrolCommentDto.java
package com.example.matchteam.dto.board.comment;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class EnrolCommentDto {
    private String commentContent;
    private String email;
    private String boardTitle;
    private String boardContent;

    @Builder
    public EnrolCommentDto(String commentContent, String email, String boardTitle, String boardContent) {
        this.commentContent = commentContent;
        this.email = email;
        this.boardTitle = boardTitle;
        this.boardContent = boardContent;
    }
}

File: MatchingTeam/app/src/main/java/com/example/matchingteam/dto/board/comment/EnrolCommentDto.kt
package com.example.matchingteam.dto.board.comment

import com.google.gson.annotations.SerializedName

data class EnrolCommentDto(
    @SerializedName("email") val email: String,
    @SerializedName("commentContent") val commentContent: String,
    @SerializedName("boardTitle") val boardTitle: String,
    @SerializedName("boardContent") val boardContent: String
)